docs/monitors/expvar.md

```diff
@@ -190,6 +190,7 @@ The **nested** `metrics` config object has the following fields:
 | `JSONPath` | **yes** | `string` | JSON path of the metric value |
 | `type` | **yes** | `string` | SignalFx metric type. Possible values are "gauge" or "cumulative" |
 | `dimensions` | no | `list of objects (see below)` | Metric dimensions |
+| `pathSeparator` | no | `string` | Metric value path separator character in JSON. The default character is / (**default:** `/`) |
```

Comment:
No need to say "The default character is /" because the doc already will this for you.